### PR TITLE
Update Shell output texts

### PR DESCRIPTION
**Pull request by DamithaSenevirathne**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md [hunk 1]
```diff
@@ -288,13 +289,11 @@ For example, see below.
...
 | 
-| Topic description :
+| # Strings
 | 
-|  The `string` type represents immutable sequence of zero or more Unicode characters. 
-|  There is no separate character type: a character is represented by a `string` of length 1.
-|  Two `string` values are `==` if both sequences have the same characters.
-|  You can use `<`, `<=`, `>`, and `>=` operators on `string` values and they work by comparing code points.
-|  Unpaired surrogates are not allowed.
+| The `string` type represents immutable sequence of zero or more Unicode characters.  There is no separate character type: a character is represented by a `string` of length 1.
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md:294]:
```suggestion
| The `string` type represents an immutable sequence of zero or more Unicode characters.  There is no separate character type: a character is represented by a `string` of length 1.
```

File: swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md [hunk 2]
```diff
@@ -288,13 +289,11 @@ For example, see below.
...
+| 
+| Two `string` values are `==` if both sequences have the same characters. You can use `<`, `<=`, `>`, and `>=` operators on `string` values and they work by comparing code points. Unpaired surrogates are not allowed.
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md:296]:
```suggestion
| Two `string` values are `==` if both sequences have the same characters. You can use `<`, `<=`, `>`, and `>=` operators on `string` values and they work by comparing the code points. Unpaired surrogates are not allowed.
```

File: swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md (hunk 1 above)
Comment (DamithaSenevirathne) [reply swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md:294]: Here we are reading the text from the BBE `.md` files. So we need to correct them in the `.md` file

File: swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md (hunk 2 above)
Comment (DamithaSenevirathne) [reply swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md:296]: same as above comment

File: swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md (hunk 1 above)
Comment (praneesha) [reply swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md:294]: BBE source is fixed via https://github.com/ballerina-platform/ballerina-distribution/pull/3340.

File: swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md (hunk 2 above)
Comment (praneesha) [reply swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md:296]: The BBE source is fixed via https://github.com/ballerina-platform/ballerina-distribution/pull/3340.

**Review by praneesha — APPROVED** [review]

**Review by mohanvive — APPROVED** [review]

**Pull request merged**